src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"           rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="icon"
+          src="./images/logo.png"
+          alt="moyo"
+        >
+      </a>
+      <nav>
+        <ul class="nav">
+          <a class="nav__link is-active" href="#">
+            Apple
+          </a>
```

Comment:
Let's rewrite it according to this guide code style
![image](https://github.com/mate-academy/layout_moyo-header/assets/64750958/ae6c3b50-5cfd-40a0-8cce-36c70256b927)